.github/CODEOWNERS

```diff
@@ -1,6 +1,4 @@
 # Lines starting with '#' are comments.
 # Each line is a file pattern followed by one or more owners.
 
-* @axelarnetwork/core
-./.github/workflows/* @talalashraf
```

Comment:
devops can be tagged manually when needed, but protocol team should own the gh workflows too I think